sys-libs/ncurses/ncurses-6.2-r1.ebuild

```diff
@@ -105,6 +111,8 @@ multilib_src_configure() {
 }
 
 do_configure() {
```

Comment:
Here as well.
How about adding a short comment, like `Flatcar: Also allow writes to /dev/ptmx, which sometimes causes the sandbox to fail Jenkins builds.`?

Reply:
Same here, I will add this too.